pkg/machines/index.es6

```diff
@@ -51,4 +51,5 @@ export function appMain() {
 
     // initiate data retrieval
     store.dispatch(getAllVms());
+    store.dispatch(getHypervisorMaxVCPU());
```

Comment:
For this purpose there's `init()` function of a provider, please move this code there.
See LIBVIRT_PROVIDER.init(context).

See `OVIRT_PROVIDER.init()` in `pkg/ovirt/provider.es6` for example how to use it.

Reply:
Important: please make sure these changes work or at least do not break `pkg/ovirt`